cfme/tests/test_appliance.py

```diff
@@ -180,3 +181,26 @@ def test_keys_included(appliance, soft_assert):
         soft_assert(appliance.ssh_client.run_command(
             "test -e /var/www/miq/vmdb/certs/{}".format(k)).success,
             "{} was not included in the build".format(k))
+
+
+@pytest.mark.uncollectif(lambda appliance: appliance.version < '5.10')
+def test_codename_in_log(configured_appliance):
+    log = '/var/www/miq/vmdb/log/evm.log'
+    configured_appliance.ssh_client.run_command('echo > {}'.format(log))
+    configured_appliance.ssh_client.run_command('appliance_console_cli --server=restart')
+
+    def codename_in_log():
+        configured_appliance.ssh_client.run_command(r'egrep "Codename: \w+$" {}'.format(log))
```

Comment:
Question and Required:
This method not returning anything. Then on what basis you are checking for `wait_for`?